app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlInteractor.kt

```diff
@@ -162,6 +163,13 @@ interface OnboardingInteractor {
      * Opens a custom tab to privacy notice url. Called when a user clicks on the "read our privacy notice" button.
      */
     fun onReadPrivacyNoticeClicked()
+
+    /**
+     * Show Wallpapers onboarding dialog to onboard users about the feature if conditions are met.
+     * Returns true if the call has been passed down to the controller.
+     * @param state The wallpaper state.
```

Comment:
This comment about returns could be formatted as an `@return` doc string. It may also be helpful to specify what the boolean return means, something like:

`@return Whether the onboarding dialog is currently shown`